### PR TITLE
refactor&feat(test, Norway): Positive rate added 

### PR DESCRIPTION
**Pull request by Rouein**
#2409

### CODE REVIEW

**Comment by lucasrodes** [comment]
LGTM! Will merge after @camappel's approval

**Review by lucasrodes — APPROVED** [review]

**Review by camappel — APPROVED** [review]
Works for me. Thanks for adding the positive rate

**Pull request merged**
